Pipfile

```diff
@@ -6,10 +6,13 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-ansible = "==2.9.9"
-molecule = "==3.0.4"
-docker = "==4.1.0"
-ansible-lint = "==4.2.0"
+ansible = "==5.2.0"
```

Comment:
Great job! There are references to versions in the REDME that should be updated.